### PR TITLE
[EPMDEPS-15617] Preparing for Labeling tool packaging

### PR DESCRIPTION
**Pull request by NatalliaHutovich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,10 @@ The config object contains all necessary settings and configurations.
 
 Check the working example of the config [here](https://git.epam.com/epm-deps/deps-LT/-/blob/main/src/config/shim.js). This config could be used for playing with LabelingTool package. 
 
+## Requirements
+
+Min NodeJS version 14.16.0
```
Comment (DrDiman) [inline README.md:121]: Min NodeJS version is 14.16.0

File: README.md [hunk 2]
```diff
@@ -116,6 +116,10 @@ The config object contains all necessary settings and configurations.
 
 Check the working example of the config [here](https://git.epam.com/epm-deps/deps-LT/-/blob/main/src/config/shim.js). This config could be used for playing with LabelingTool package. 
 
+## Requirements
+
```
Comment (DrDiman) [inline README.md:120]: remove empty line

**Review by DrDiman — APPROVED** [review]

**Comment by Rob-n-Dude** [comment]
👍

File: README.md (hunk 1 above)
Comment (NatalliaHutovich) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (NatalliaHutovich) [reply README.md]: Removed

**Pull request merged**